[2021-09-20 14:35:36,519] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: branching-dag.Conditional_Select 2020-07-25T00:00:00+00:00 [queued]>
[2021-09-20 14:35:36,530] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: branching-dag.Conditional_Select 2020-07-25T00:00:00+00:00 [queued]>
[2021-09-20 14:35:36,530] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-09-20 14:35:36,531] {taskinstance.py:1095} INFO - Starting attempt 3 of 4
[2021-09-20 14:35:36,532] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-09-20 14:35:36,543] {taskinstance.py:1114} INFO - Executing <Task(BranchPythonOperator): Conditional_Select> on 2020-07-25T00:00:00+00:00
[2021-09-20 14:35:36,547] {standard_task_runner.py:52} INFO - Started process 1197 to run task
[2021-09-20 14:35:36,550] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'branching-dag', 'Conditional_Select', '2020-07-25T00:00:00+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_branching.py', '--cfg-path', '/tmp/tmpz0gesmvv', '--error-file', '/tmp/tmpnsvzxqm5']
[2021-09-20 14:35:36,553] {standard_task_runner.py:77} INFO - Job 11: Subtask Conditional_Select
[2021-09-20 14:35:36,593] {logging_mixin.py:109} INFO - Running <TaskInstance: branching-dag.Conditional_Select 2020-07-25T00:00:00+00:00 [running]> on host ba08e7b1fb26
[2021-09-20 14:35:36,631] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=myemail@mail.com
AIRFLOW_CTX_DAG_OWNER=rab4u
AIRFLOW_CTX_DAG_ID=branching-dag
AIRFLOW_CTX_TASK_ID=Conditional_Select
AIRFLOW_CTX_EXECUTION_DATE=2020-07-25T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-07-25T00:00:00+00:00
[2021-09-20 14:35:36,639] {logging_mixin.py:109} INFO - selecting task 1
[2021-09-20 14:35:36,640] {python.py:151} INFO - Done. Returned value was: prod
[2021-09-20 14:35:36,641] {skipmixin.py:124} INFO - Following branch prod
[2021-09-20 14:35:36,653] {taskinstance.py:1462} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1164, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1282, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1312, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 180, in execute
    self.skip_all_except(context['ti'], branch)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/skipmixin.py", line 150, in skip_all_except
    branch_task_ids.update(dag.get_task(branch_task_id).get_flat_relative_ids(upstream=False))
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dag.py", line 1547, in get_task
    raise TaskNotFound(f"Task {task_id} not found")
airflow.exceptions.TaskNotFound: Task prod not found
[2021-09-20 14:35:36,656] {taskinstance.py:1505} INFO - Marking task as UP_FOR_RETRY. dag_id=branching-dag, task_id=Conditional_Select, execution_date=20200725T000000, start_date=20210920T143536, end_date=20210920T143536
[2021-09-20 14:35:36,683] {local_task_job.py:151} INFO - Task exited with return code 1
[2021-09-20 14:35:36,704] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
